Base_Scripts/Scrapers/template/list_pdf_extractor/configs.py

```diff
@@ -0,0 +1,19 @@
+# Set to the page you want to scrape
```

Comment:
So what you've got going with these configs is interesting to me. These don't really need to be Python, they could just be JSON or YAML files that you'd parse with the relevant library, but that got me thinking:

Instead of making a ton of small python programs that import these configs and such, could you have a single program that does everything from a config file? Then you'd just call `python extract_pdf.py ./CA/alameda.yaml` or something and that'd do everything. It'd be less code duplication and may address the wonky import issues.

Reply:
I was thinking about simply having all the code within in the configs, as it's now only a few lines long, but I wasn't sure if it would help with the setup at all.